src/components/button/Button.vue

```diff
@@ -242,6 +250,10 @@ export default {
             }
         }
 
+        &.vs-button--rounded {
+            border-radius: 100px;
```

Comment:
Can this be updated to use a token? We have a border-radius.yml token file. These tokens will need reviewed when we review tokens but for now we should keep things consistent.

Reply:
@GemmaLouise I've replaced it with `border-radius-pill`